packages/bot-web-ui/src/components/load-modal.jsx

```diff
@@ -25,25 +25,28 @@ const LoadModal = ({
 
     if (is_mobile) {
         return (
-            <FadeWrapper is_visible={is_load_modal_open} className='load-strategy__wrapper' keyname='save__wrapper'>
-                <PageOverlay header={localize('Load strategy')} onClickClose={toggleLoadModal}>
-                    <Div100vhContainer className='load-strategy--is-mobile' height_offset='80px'>
-                        <Tabs
-                            active_index={active_index}
-                            onTabItemClick={setActiveTabIndex}
-                            should_delay_render={should_rerender_tabs}
-                            top
-                        >
-                            <div label={localize('Local')}>
-                                <Local />
-                            </div>
-                            <div label='Google Drive'>
-                                <GoogleDrive />
-                            </div>
-                        </Tabs>
-                    </Div100vhContainer>
-                </PageOverlay>
-            </FadeWrapper>
+            <MobileFullPageModal
+                is_modal_open={is_load_modal_open}
+                className='load-strategy'
+                header={localize('Load strategy')}
+                onClickClose={toggleLoadModal}
+                offset='80px'
```

Comment:
```suggestion
                height_offset='80px'
```

Just calling it `offset` is a bit verbose.